src/main/java/com/laa/model/DecisionReport.java

```diff
@@ -1,9 +1,6 @@
 package com.laa.model;
 
 public abstract class DecisionReport {
-	
```

Comment:
why the spaces here?

Reply:
there were more spaces, i tried to reduce , i have reduced it further